package.json

```diff
@@ -28,11 +28,13 @@
     "test"
   ],
   "bin": {
-    "reskindex": "scripts/reskindex.js"
+    "reskindex": "scripts/reskindex.js",
+    "matrix-gen-i18n": "scripts/gen-i18n.js"
```

Comment:
Why the `matrix-` here ooi?

Reply:
If you (for some reason) npm install matrix-react-sdk with the `-g` flag, it'll get installed globally, so I figured it ought to be more specific.